cdap-ui/app/features/adapters/templates/tabs/datasets.html

```diff
@@ -0,0 +1,17 @@
+<div class="table-responsive">
+  <table class="table table-bordered">
```

Comment:
We should have a view where is there are no associated datasets (before even running once)